tests/Utilities/MakeWithRandomValues.hpp

```diff
@@ -73,67 +89,40 @@ struct FillWithRandomValuesImpl<Variables<tmpl::list<Tags...>>,
 }  // namespace TestHelpers_detail
 
 /// \ingroup TestingFrameworkGroup
-/// \brief A uniform distribution functor for which the behavior can be easily
-/// customized
 ///
-/// \details The two constructors take either a low and high bound arguments or
-/// a similar 2-component array , which can be passed in in any type from which
-/// the generated type can be constructed. The default generation function
-/// operates as would be expected for a double
+/// \brief A uniform distribution functor which redirects appropriately to
+/// either the `std::uniform_int_distribution` or the
+/// `std::uniform_real_distribution` as appropriate. also provides a convenience
+/// constructor which takes a 2-element array for the bounds.  either floating
+/// point or int distributions
+// {@
+template <typename T, typename Enable = std::nullptr_t>
+class UniformCustomDistribution {};
 template <typename T>
-struct UniformCustomDistribution {
- private:
-  std::array<T, 2> bounds_;
-
+class UniformCustomDistribution<T, Requires<std::is_integral<T>::value>>
```

Comment:
could you add to `Utilities/TypeTraits.hpp` `cpp17::is_integral_v`? Just follow the examples of `is_floating_point`. Thanks!